Changes to key guide (#194)
meilisearch/documentation: guides/advanced_guides/keys.md

### Reset keys

Since the private and public keys are generated based on your master key, you must change the master key to change the two other ones.

This means that you will have to restart MeiliSearch with the new master key.

> **All key will be changed**. This means that a single key can not be changed.

Reviewer: 2 times this means. If you can remove one it's cool, if not, we will see that later

Author: Changed !

Since the private and public keys are generated based on your master key, you must change the master key to change the two other ones.

> MeiliSearch needs to be restarted with the new master key for the keys to be changed.

Reviewer: What do you think?
```suggestion
After changing your master key, you must restart the MeiliSearch server to ensure the other keys renewal.
```
